### DISCUSSION TITLE
Unable to Install Debian 12

### DISCUSSION THREAD

**Discussion by SimonBatesLinux** [OpenTabletDriver/OpenTabletDriver#3188] [Q&A] [answered]
I tried to install on Debian 12 today and got the following output:
Reading package lists... Done
Building dependency tree... Done
Reading state information... Done
Note, selecting 'opentabletdriver' instead of './OpenTabletDriver.deb'
Some packages could not be installed. This may mean that you have
requested an impossible situation or if you are using the unstable
distribution that some required packages have not yet been created
or been moved out of Incoming.
The following information may help to resolve the situation:

The following packages have unmet dependencies:
 opentabletdriver : Depends: dotnet-runtime-6.0 but it is not installable
E: Unable to correct problems, you have held broken packages.

I also tried to install the runtime following: https://learn.microsoft.com/en-us/dotnet/core/install/linux-debian
Is it possible to install this driver on Debian and if so, how do I install it on debian?

**Comment by jamesbt365** [comment, marked as answer]
Install microsoft repo, `sudo apt update`, then install OTD.

**Reply by SimonBatesLinux** [reply]
That did it. Thank you.
